astropy/time/core.py

```diff
@@ -2270,7 +2270,7 @@ class ScaleValueError(Exception):
     pass
 
 
-def _make_array(val, copy=False):
+def _make_array(val, copy=False, restrict_types=True):
```

Comment:
This new argument does not seem to be used at all!

Reply:
Sorry, it's been a long day.